.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.1"
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_DB: circuitverse_development
+      POSTGRES_PASSWORD: postgres
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+      - redis
+    environment:
+      REDIS_URL: "redis://redis:6379/0"
+      CIRCUITVERSE_USE_SOLR: "false"
+      DOCKER_ENVIRONMENT: 'true'
+    command: sleep infinity
```

Comment:
Why do we need this command?

Reply:
As we are using docker-compose with devcontainer, The primary process will attach to the service after it is created. Devcontainer will attach to the service and run the scripts and manage it. To prevent it from terminating before it can attach,  we are using `sleep infinity`.

Reply:
@tachyons also official documentation suggests to use some sleep command to prevent it from terminating
https://code.visualstudio.com/docs/devcontainers/create-dev-container#_extend-your-docker-compose-file-for-development